packages/valory/contracts/service_staking_token/contract.py

```diff
@@ -19,28 +19,38 @@
 
 """This module contains the class to connect to the `ServiceStakingTokenMechUsage` contract."""
 
+from enum import Enum, auto
+
 from aea.common import JSONLike
 from aea.configurations.base import PublicId
 from aea.contracts.base import Contract
 from aea.crypto.base import LedgerApi
 
 
+class StakingState(Enum):
+    """Staking state enumeration for the staking."""
+
+    UNSTAKED = auto()
+    STAKED = auto()
+    EVICTED = auto()
+
+
 class ServiceStakingTokenContract(Contract):
     """The Service Staking contract."""
 
     contract_id = PublicId.from_str("valory/service_staking_token:0.1.0")
 
     @classmethod
-    def is_service_staked(
+    def get_service_staking_state(
         cls,
         ledger_api: LedgerApi,
         contract_address: str,
         service_id: int,
     ) -> JSONLike:
         """Check whether the service is staked."""
         contract_instance = cls.get_instance(ledger_api, contract_address)
-        res = contract_instance.functions.isServiceStaked(service_id).call()
-        return dict(data=res)
+        res = contract_instance.functions.getServiceStakingState(service_id).call()
+        return dict(data=StakingState(res))
```

Comment:
This is not returning the correct value. It returns StakingState.UNSTAKED for service ID 275, which you can verify it is staked here:
https://gnosisscan.io/address/0xf8a415d761b1279B0f1ACE68cD85e5ca4346Cc3F#readContract

Reply:
be6749a35163b8c4aa752a0c003e19eb3006b29f